app/controllers/application_controller.rb

```diff
@@ -10,7 +10,20 @@ class ApplicationController < ActionController::Base
     strategy DecentExposure::StrongParametersStrategy
   end
 
+  private
+
   def require_gds_editor_permissions!
     authorise_user!("GDS Editor")
   end
+
+  # FIXME: clean this up when we're using content_ids in the URL.
+  def find_topic_for_sector_id
```

Comment:
Would this make sense to be a method on the Topic model?

Or perhaps even on the Tag model, since I think looking up Tags by path is likely to be wanted, too.